app/_locales/en/messages.json

```diff
@@ -1033,6 +1033,12 @@
     "message": "$1 has suggested this price.",
     "description": "$1 is url for the dapp that has suggested gas settings"
   },
+  "dappSuggestedHigh": {
+    "message": "Site suggested"
+  },
```

Comment:
Q: where is `dappSuggestedHigh` being used? I see `t(title)` exists where `title` can be `dappSuggestedHighShortLabel`, but I'm having trouble finding where `dappSuggestedHigh` is used

Reply:
It's used in ui/hooks/gasFeeInput/useGasFeeInputs.js:187

Reply:
I'm still a bit confused. Those lines are
```
      if (Number(fee) > Number(GAS_LIMIT_TOO_HIGH_IN_ETH)) {
        setEstimateUsed(PriorityLevels.dappSuggestedHigh);
      } else if (transaction?.userFeeLevel) {
        setEstimateUsed(transaction?.userFeeLevel);
      }
```

I'm not seeing where the translation is used.  

I also see ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js:28-29: 
```
  } else if (priorityLevel === PriorityLevels.dappSuggestedHigh) {
    title = 'dappSuggestedHighShortLabel';
```
mentioned in the first comment which uses `dappSuggestedHighShortLabel` but not `dappSuggestedHigh`. Is there another location you could help point to?

Reply:
Oh sorry, I see now.

ui/components/app/edit-gas-fee-button/edit-gas-fee-button.js:40
ui/components/app/edit-gas-fee-button/edit-gas-fee-button.js:74

Reply:
ooh, thank you! 🙏